### DISCUSSION TITLE
Busco equipo 🤗

### DISCUSSION THREAD

**Discussion by ShaneMS2098** [Launch-X-Latam/MisionFrontEnd#547] [Team Posts]
Aún no tengo equipo si alguien aún tiene una vacante aún, me puede escribir a mi correo.
lx09872@innovaccion.mx

**Comment by renatocoello** [comment]
Yo tampoco tengo grupo y si formamos uno y vamos ingresando a los que podamos te parece, no se si armemos un discord para irnos comunicando
Mi usuario es: ***@***.***
________________________________
De: Shantal Silva ***@***.***>
Enviado: jueves, 10 de noviembre de 2022 20:20
Cc: Subscribed ***@***.***>
Asunto: [Launch-X-Latam/mision-backend] Busco equipo 🤗 (#83)


Aún no tengo equipo si alguien aún tiene una vacante aún, me puede escribir a mi correo.
***@***.******@***.***>

**Comment by mixtlistar00** [comment]
buenas horas en que lean esto.

me puedo unir a su equipo?
 muchas gracias

**Comment by JosueDuenas1995** [comment]
me puedo unir, de antemano, gracias

**Comment by renatocoello** [comment]
Quien crea el grupo para unirnos los que podemos en si de cuantas personas son máxima en cada grupo



Enviado desde mi Galaxy



-------- Mensaje original --------
De: JOSUÉ DUEÑAS BOLAÑOS ***@***.***>
Fecha: 10/11/2022 20:48 (GMT-05:00)
A:
CC: Renato Adrian Coello ***@***.***>, Comment ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] Busco equipo 🤗 (#83)


me puedo unir, de antemano, gracias

**Comment by AlexG0712** [comment]
> Aún no tengo equipo si alguien aún tiene una vacante aún, me puede escribir a mi correo.
> lx09872@innovaccion.mx

Si gustan me faltan dos integrantes, mi correo para ponernos de acuerdo
 lx04297@innovaccion.mx

**Comment by ShaneMS2098** [comment]
Muchas gracias ya me unieron a un grupo 😺
